engine/player/unique_gear_thewarwithin.cpp

```diff
@@ -6919,6 +6919,239 @@ void zees_thug_hotline( special_effect_t& effect )
+// 443535 On Use
+// 443534 On Use Buff, radiance, double all effects
+// 450696 50 verses of Resillience tracking buff
+// 450699 50 verses of Radiance tracking buff
+// 450706 Inner Resillience Proc driver
+// 450720 Inner Radiance Proc driver
+// 450719 Magic Absorb buff
+// 450721 Ire of Devotion damage spell
+
+// 443533 Values
+// s1 - static value of 50
+// s2 - crit strike amount increase - Used in 450720 buff
+// s3 - devotion of ire damage - Used in 450721 spell
+// s4 - armor increase - Used in 450706 buff
+// s5 - magic absorb value - Used in 450719 buff
+void tome_of_lights_devotion( special_effect_t& effect )
+{
+  auto equip_driver = effect.player->find_spell( 443533 );
+  assert( equip_driver && "Tome of Lights Devotion equip driver not found" );
+
+  // Setup double value buff
+  auto radiance_buff = create_buff<buff_t>( effect.player, effect.player->find_spell( 443534 ) );
+
+  struct inner_resilience_cb_t : public dbc_proc_callback_t
+  {
+    struct ward_of_devotion_buff_t : public absorb_buff_t
+    {
+      ward_of_devotion_buff_t( const special_effect_t& e, const spell_data_t* s, const spell_data_t* data )
+        : absorb_buff_t( e.player, "ward_of_devotion", s )
+      {
+        cumulative = false;
+        set_default_value( data->effectN( 5 ).average( e ) );
+      }
+    };
+    buff_t* ward_of_devotion_buff;
+
+    const spell_data_t* data;
+
+    inner_resilience_cb_t( const special_effect_t& e )
+      : dbc_proc_callback_t( e.player, e ),
+      ward_of_devotion_buff( nullptr )
+    {
+      data = e.player->find_spell( 443533 );
+      ward_of_devotion_buff = make_buff<ward_of_devotion_buff_t>( e, e.player->find_spell( 450719 ), data );
+    }
+
+    void execute( action_t*, action_state_t* s ) override
+    {
+      ward_of_devotion_buff->trigger();
+    }
+  };
+
+  struct tome_of_lights_devotion_cb_t : public dbc_proc_callback_t
+  {
+    bool resilience_phase;
+    dbc_proc_callback_t* inner_resilience_cb;
+    dbc_proc_callback_t* inner_radiance_cb;
+
+    // Setup both tracking buffs
+    buff_t* resilience_verses_tracker;
+    buff_t* radiance_verses_tracker;
+
+    // Setup the armor and crit buffs
+    buff_t* inner_radiance_buff;    // Crit buff
+    buff_t* inner_resilience_buff;  // Armor buff
+
+    // Pointer to radiance buff
+    buff_t* radiance_buff;
+
+    struct ire_of_devotion_t : generic_proc_t
+    {
+      buff_t* radiance_buff;
+      ire_of_devotion_t( const special_effect_t& e, const spell_data_t* spell, const spell_data_t* data, buff_t* rb )
+        : generic_proc_t( e, "ire_of_devotion", spell ),
+          radiance_buff( nullptr )
+      {
+        radiance_buff = rb;
+        base_dd_min = base_dd_max = data->effectN( 3 ).average( e.item );
+      }
+
+      double composite_da_multiplier( const action_state_t* s ) const override
+      {
+        double m = generic_proc_t::composite_da_multiplier( s );
+
+        if ( radiance_buff->check() )
+            m *= 2;
```

Comment:
Is this value not contained in spell data?

Reply:
sadly no.  Just says increased effectiveness.  Value was found via manual testing.